cg/services/orders/validation/order_types/rna_fusion/models/sample.py

```diff
@@ -21,3 +21,21 @@ class RNAFusionSample(Sample):
     source: str
     subject_id: str = Field(pattern=NAME_PATTERN, min_length=1, max_length=128)
     tissue_block_size: TissueBlockEnum | None = None
+    tumour: bool = True
+
+    @field_validator("tumour")
```

Comment:
Getting a warning here: "Decorator will not receive the callable it may expect". Probably will ignore as all field_validators in cg rais ethe same warning